### DISCUSSION TITLE
It's impossible

### DISCUSSION THREAD

**Discussion by sirhaffy** [community/community#139173] [GitHub Education]
### Hi everyone,

I have now tried like 15 times to get access to Education, I am giving up..

I have got a new certificate from the school showing it's a hybrid course throug virtual classroom (Google Classroom). Did not help. My teacher has reached out to GitHub, but no real response.

When I use my computers 720 camera. The image is very bad. So I try to use my iPhone 13 Pro and iPad Pro, the only camera that works is the UltraWide Frontal Camera. The others don't work, maby a bug on tha page.. And those cameras give very bad ressults scanning documents.

I tried to fake it all with a DesktopRecorder > WebCam software. Did not work eather.

I do not know what to do, I have opened two different support cases, but GitHub does not give me any help. 

What documents are they asking for? Seems like the Swedish ones I provide does not cut it (Mekenat Student ID and Certificate from the school).

I have done everything they ask for, like verify school e-mail, correct name in profile and such..

This is what I get as response.

- Please use your device camera to submit your academic affiliation documents. If you’re having trouble using your device camera, please allow camera access for the application website on your browser. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found here. You may also try applying from a different computer or mobile device.

I have..

- The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.

It does, multiple times..

- You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

I have provided the certificate that explains that it is a virtual classroom (google classroom) that is used for student's far from school.

**Comment by hkqr** [comment]
Hello @sirhaffy , Letme reply ur issue problem.

Potential solutions:

Double-check that your documents meet all the requirements specified by GitHub Education, including:
The document must contain your complete school name, not just the logo.
If the document is not in English, you need to provide an English translation.
The document must indicate virtual learning if you're remote.
Ensure that you've allowed camera access for the GitHub Education website on your browser.
Try submitting your documents again, making sure to include all the required information.
If you're still having trouble, consider reaching out to GitHub Education support again and providing more detailed information about your virtual course and the documents you've submitted.


Additional suggestions:

You may want to try using a different browser or device to see if that resolves the issue.
If you're using a VPN, try disabling it to see if that makes a difference.
Consider providing more context or information about your virtual course and the documents you've submitted in your support ticket, as this may help GitHub Education support better understand your situation.

**Reply by sirhaffy** [reply]
Yes, all of those things I have checked. That's why I am here now.. Hoping someone from GitHub could help me. As when I connect to their support there is no option for "Education" as I can find.

**Comment by mymh13** [comment]
Hello, I have exactly the same issues as @sirhaffy 

We have the same school document. I have tried to supply pictures of it both in Swedish and transted into English through googles .pdf translation service. Let us check at the list you supplied @opwx :
1) The document contains complete school name (four times)
2) As mentioned, we get the chance to take two pictures in the application, I have supplied both English and Swedish versions, many times
3) The document clearly states "The training is conducted as a hybrid, i.e. both remotely via Google Meet (virtual teaching) and on site in Mölndal"
4) Camera is allowed, there is no problem taking the picture, the problem is trying to photograph an image on the screen (I have no printer) from the mobile phone. I have no clue what it looks like on your end, but it gets grainy - and for some reason it uses wide/fisheye so the text bends (that is not my choice, that is your system! What is wrong with uploading the file only?)
5) I've been rejected I believe four times, and I have also escalated this through tickets as well as included the document in a .pdf in my tickets (but support does not even look at the attached document)
6) I've sent ticket, they just state "it should work now, try again", which I did and by default it was auto-rejected within a few seconds

Not using any VPN

Regards,
Niklas

**Reply by ItaiBEdu** [reply]
I have the same issues. My phone's camera is excellent, but whenever I use it via the GitHub application form, the picture is very grainy. It feels as though GitHub actively sabotages my application.

I opened several tickets and the replies I get are generic. Feels like AI-generated.

Very frustrating.

**Comment by sirhaffy** [comment]
These are the documents I have uploaded:

![Screenshot 2024-09-20 131301](https://github.com/user-attachments/assets/bc26abb9-b563-4db7-9ef3-ec679bcac86c)
I have also made an translation of the certificate with google translate, did not work ether.

![Screenshot 2024-09-20 131327](https://github.com/user-attachments/assets/14dd6247-c8e1-470c-a0ca-3b9198cc0652)
This is my student ID.

**Comment by mymh13** [comment]
I tried to apply again, and this is the email I get in return:

![image](https://github.com/user-attachments/assets/3d5da916-e6a1-4961-b957-c8f594fb5df0)

**Comment by mymh13** [comment]
This is the document I am uploading. Let us break down the points listed above:
1) "It does not contain your school name" - It does.
2) "contains your last name exactly as it appears in Billing info" - It does.
3) "Date" - I sent in two pictures: one displaying the Swedish next to the English, one showing just one of the certificates. You have the info.
4) "Not near any campus location" - no, because this is a hybrid program, as listed in the certificate AND in my application.
5) "contains first name" - It does.
6) "GitHub user profile" - It has the exact same name with the exact same spelling as in the documents.

It's quite offensive to be told to send in information that I already sent in. I do what is asked of me and still I get rejected and scolded? Fantastic service, this.

![github issues](https://github.com/user-attachments/assets/e3a26d44-9e11-47f8-add5-af574ee1e5f3)

**Reply by JoseGonz11** [reply]
I just sent an apply with an image of a map using OBSStudio-VirtualCamera (Screenshots, but way jankier) but the image is only about 700x500 (inspected the blob) wich is ultra blurry, so no readable text, w/ phone is no chance to be readable even with 4K camera as pixels in my TV are too big for such low quality. I am inputting a 1920x1080 image from screenshots and 4K from phone. Also I am in a distance only new official format on Andalucia, Spain, so no shot if no human gets the full quality proofs.

**Reply by ccn8-caleb** [reply]
Is it just me or is this bad by design? Why else would they ask for non-blurry images and then force us to use a camera and compress the image to ensure text is difficult to read?

I have also had trouble with my information, even after multiple verifications of distance/virtual/online learning. Not being able to get past the automated application denier is really frustrating.

**Comment by hilda0618** [comment]
I am having the same issue where it doesn't allow me to even upload the image and verification. It is very frustrating.

**Comment by mymh13** [comment]
I solved this issue eventually, but it not an option for everyone to do what I did unfortunately:

1. I moved to the schools location - at our school I could upload files while at home I had to take a picture.
2. I uploaded a picture with two images in one: our certificate in original (local language) to the left, and a translation pasted in on the side to the right.

The real problem is that GitHub does not allow us to upload images, as soon as I could do that the issue was resolved quite quickly. Using a camera is a terrible option. Or rather, not an option.

**Comment by IonutInit** [comment]
For me it doesn't work with proof of enrolment to a 3-year course at a UK university. And having the camera through a browser as the only option to upload documents is absolutely ridiculous. I suppose Microsoft doesn't want doesn't want to give out too many freebies.

You did learn something through all this, they're called dark patterns ;)

**Comment by Glue-cose** [comment]
I just solved the 'having trouble using your device camera' problem by resending the requset with a different browser (Mozilla Firefox).
